docs/publishing-to-app-store.md

```diff
@@ -48,10 +48,10 @@ The static bundle is built every time you target a physical device, even in Debu
 
 ### 3. Build app for release
 
-You can now build your app for release by tapping `⌘B` or selecting **Product** → **Build** from the menu bar. Once built for release, you'll be able to distribute the app to beta testers and submit the app to the App Store.
+You can now build your app for release by tapping `⌘ + B` or selecting **Product** → **Build** from the menu bar. Once built for release, you'll be able to distribute the app to beta testers and submit the app to the App Store.
 
 :::info
-You can also use the `React Native CLI` to perform this operation using the option `--mode` with the value `Release` (e.g. `npx react-native run-ios --mode Release`).
+You can also use the `React Native CLI` to perform this operation using the option `--mode` with the value `release` (e.g. from the root of your project: `npm run ios -- --mode="release"` or `yarn ios --mode release`).
```

Comment:
This note was a subject of rejected PR lately, are you sure that changing the casing of the mode is fine?

Reply:
## Android

The RN CLI docs suggests **"_lowercase_"** for _Android_: https://github.com/react-native-community/cli/blob/main/packages/cli-platform-android/README.md#--mode-string

defaults to `debug` (lowercase): https://github.com/react-native-community/cli/blob/4650a8d41c5f9851e0f6bf5939a1bd8f4326c990/packages/cli-platform-android/src/commands/runAndroid/getTaskNames.ts#L8

also, while installing for Android, CLI applies **`.toLowecase()`** to `mode` value

https://github.com/react-native-community/cli/blob/4650a8d41c5f9851e0f6bf5939a1bd8f4326c990/packages/cli-platform-android/src/commands/runAndroid/tryInstallAppOnDevice.ts#L19

## iOS

However, _iOS_ docs mention it's **"_case-sensitive_"** and should have _first letter capitalized_:
https://github.com/react-native-community/cli/blob/main/packages/cli-platform-ios/README.md#--mode-string

## 🤔

### Pushing changes to follow iOS docs 😐

P.S.: I had tested only the Android commands since I use Win x64, but it seems we might need to change the iOS commands.

P.S.: On a side note, CLI should keep this **_case-insensitive_** only or should internally use a map or enum with iOS specs if any, most of the standard cli programs work in a case-insensitive way.

---

Btw, nice catch @Simek :)